charts/datadog/templates/_daemonset-volumes-linux.yaml

```diff
@@ -74,14 +74,16 @@
     path: /etc/passwd
   name: passwd
 {{- end }}
+{{- if or (and (eq (include "should-enable-system-probe" .) "true") .Values.datadog.serviceMonitoring.enabled) (and (eq (include "should-enable-security-agent" .) "true") .Values.datadog.securityAgent.compliance.enabled) }}
```

Comment:
IMO the PR should update the function `system-probe-feature` call by `should-enable-system-probe` to also include `.Values.datadog.serviceMonitoring.enabled` (if `networkMonitoring.enabled==true` is not mandatory to use `serviceMonitoring`)

Reply:
The helper was updated to include that. I think @p-lambert just needs to remove the `.Values.datadog.serviceMonitoring.enabled` part.

Reply:
I don't see `serviceMonitoring` used in `system-probe-feature` on this PR or on the main branch: https://github.com/DataDog/helm-charts/blob/main/charts/datadog/templates/_helpers.tpl#L258

Reply:
Isn't this it? https://github.com/DataDog/helm-charts/pull/437/files#diff-483dad0ace7ed469efb9d93ba2f3d3c2045bbc1cd317f49e7e0a122bdf9d6116R258

Reply:
AH, yeah, sorry about that. This file was hide in my browser